finals-2024-01-15/code/5100-nenad-saldziev/src/app/addons/movie.ts

```diff
@@ -0,0 +1,52 @@
+export interface movie{
```

Comment:
+5 points for the models, the definitions are ok. Note that in typescript `[string]` and `string[]` are used for different things.